images/rabbitmq-cluster/cluster-rabbit.sh

```diff
@@ -13,5 +13,12 @@ echo NODENAME=rabbit@${HOSTNAME}.${SERVICE_NAME}-headless.${POD_NAMESPACE}.svc.c
 echo cluster_formation.k8s.hostname_suffix=.${SERVICE_NAME}-headless.${POD_NAMESPACE}.svc.cluster.local >> /etc/rabbitmq/rabbitmq.conf
 echo cluster_formation.k8s.service_name=${SERVICE_NAME}-headless >> /etc/rabbitmq/rabbitmq.conf
 
+# Replace placeholders with ENV values and copy file to rabbitmq conf directory
+if [ -e /etc/rabbitmq/definitions.json ]; then
+  sed -i -e "s/REPLACE_USERNAME/${RABBITMQ_DEFAULT_USER}/g" \
```

Comment:
we already use envplate: https://github.com/amazeeio/lagoon/blob/484d8c9ecf6b227587a015bb96d25398ca4a9500/images/commons/Dockerfile#L16 example: https://github.com/amazeeio/lagoon/blob/master/images/php/fpm/entrypoints/70-php-config.sh#L4 for needs of replacing env variables inside files. Let's use the same system also here, so we are consistent across Lagoon.

Reply:
Added envplate binary and set variables in definitions.json